nixos/modules/services/web-apps/nextcloud-notify_push.nix

```diff
@@ -85,9 +82,10 @@ in
           else
             "@${cfg.dbhost}");
         dbName = lib.optionalString (cfg.dbname != null) "/${cfg.dbname}";
-        dbUrl = "${dbType}://${dbUser}${dbPass}${lib.optionalString (!isSocket) dbHost}${dbName}${lib.optionalString isSocket dbHost}";
-      in lib.optionalString (dbPass != "") ''
-        export DATABASE_PASSWORD="$(<"${cfg.dbpassFile}")"
+        dbUrl = "${dbType}://${dbUser}$dbPass${lib.optionalString (!isSocket) dbHost}${dbName}${lib.optionalString isSocket dbHost}";
+      in lib.optionalString (dbHost != "") ''
+        export DATABASE_PASSWORD="$(${lib.getExe pkgs.jq} -r '.dbpass // ""' "${toString cfgN.secretFile}")"
```

Comment:
If we read from Nextcloud's secret file, there are a few things to consider:
* can this service even read from that file?
* I assume that the options above are redefined to allow deploying this to a different machine, but using the secretFile here again kinda defeats this purpose. OTOH there's `cfgN` re-using Nextcloud settings, but then duplicating some options here doesn't make much sense.

anyways, I'd like to defer to @SuperSandro2000 for that, I'm neither a maintainer nor a user of this module.

Reply:
I am using notify_push on the same machine and connect to postgres via unix socket 😅

Reply:
@SuperSandro2000 would you be fine if we duplicating the database config options for notify_push module?